src/utils.js

```diff
@@ -21,7 +21,7 @@ const globby = require('globby')
 const path = require('path')
 const archiver = require('archiver')
 // this is a static list that comes from here: https://developer.adobe.com/runtime/docs/guides/reference/runtimes/
-const SupportedRuntimes = ['nodejs:10', 'nodejs:12', 'nodejs:14']
+const SupportedRuntimes = ['nodejs:10', 'nodejs:12', 'nodejs:14', 'nodejs:16']
```

Comment:
Do you have a strategy when to stop supporting node versions?

Reply:
The strategy is that we follow what the Runtime server supports. Actually, we could improve it here by querying the server metadata (and cache the response after the first time) so this list is dynamic...

Reply:
related: https://github.com/adobe/aio-lib-runtime/pull/101

Reply:
#101 is merged, it is a good utility function.
I would prefer that we do not check the server for supported runtimes on every action deploy though.  Can we do it in response to an error as a way to provide more details to the user?

Reply:
Good idea - we can output that only on error. 

That is one case - if they are running node-10 and the server doesn't support it anymore, we output the supported runtimes.

The other case - if our action is node-18, and the server supports it, but the CLI code hasn't been updated. Do we want to support a way to "override" this check? I would think we would quickly update the CLI code so this wouldn't really be an issue but perhaps node-18 is in a testing phase or something and we don't want to make it generally available. Maybe an env var?

Reply:
Note for the second case, if the server doesn't support node-10 and they override, the server will not accept it anyway on create/update via the API)

Reply:
I'll skip the override for now and just implement calling `getSupportedRuntimes` on error.

Reply:
done, see latest commit